code/game/objects/items/cigs_lighters.dm

```diff
@@ -265,13 +268,17 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 		reagents.remove_any(to_smoke)
 
 /obj/item/clothing/mask/cigarette/process(delta_time)
-	var/turf/location = get_turf(src)
+	var/turf/open/O = get_turf(src)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Reply:
You *removed* the good name from it 🥴

Reply:
Fixed with 2207989.  Eh, I think having two variables named loc and location is bad.  The O was taken from another part of the code I was looking at.